packages/ipfs-core/src/components/pubsub.js

```diff
@@ -41,6 +41,7 @@ module.exports = ({ network, config }) => {
    */
   async function subscribe (topic, handler, options) {
     const { libp2p } = await network.use(options)
+    // @ts-ignore Libp2p Pubsub is deprecating the handler, using the EventEmitter
```

Comment:
At some point we might get https://github.com/libp2p/js-libp2p/blob/0.30.x/src/pubsub-adapter.js this out. I can add in a follow up PR the handlers behaviour for IPFS